Manual/SSDT-PNLF.dsl

```diff
@@ -235,7 +245,7 @@ DefinitionBlock("", "SSDT", 2, "ACDT", "PNLF", 0)
                 If (!(8 & Local4) && Local2 != Local1)
                 {
                     // set new backlight PWMAX but retain current backlight level by scaling
-                    Local0 = (((^LEVX & 0xFFFF) * Local2) / Local1) | (Local2 << 16)
+                    Local0 = (((^LEVX & 0xFF3E) * Local2) / Local1) | (Local2 << 16)
```

Comment:
This change is wrong. This line of code is for Haswell to Kaby Lake. On these platforms, both backlight max frequency and duty cycle are controlled using only one 32-bit register: low 16 bits are for duty cycle and high 16 bits are for max frequency.

`^LEVX & 0xFFFF` gives us the low 16 bits, or duty cycle. We then rescale this value to match the new PWMMAX being set.